.travis.yml

```diff
@@ -84,3 +84,7 @@ deploy:
         condition: $TRAVIS_OS_NAME = linux
         tags: true
         repo: onivim/oni
+    - provider: script
+      script: build/aur/deploy.sh
+      on:
+          tags: true
```

Comment:
We'll still need this condition here:
`condition: $TRAVIS_OS_NAME = linux`

Otherwise OSX builds for tagged commits will fail, since they will try and run the AUR deploy script too. If we add that condition line, that will gate the provider so the script will only run on Linux